Author: Bertrand Marc <bmarc@debian.org>
Description: the file testbed_test.c does not exist anymore, so this is
 temporarily removed from the documentation.

--- a/doc/documentation/gnunet-c-tutorial.texi
+++ b/doc/documentation/gnunet-c-tutorial.texi
@@ -694,12 +694,6 @@
 about them can be found in the testbed default configuration file
 @file{src/testbed/testbed.conf}.
 
-With the testbed API, a sample test case can be structured as follows:
-
-@example
-@verbatiminclude testbed_test.c
-@end example
-
 @noindent
 The source code for the above listing can be found at
 @uref{https://gnunet.org/git/gnunet.git/tree/doc/
